jujugui/static/gui/src/app/init/utils.js

```diff
@@ -105,6 +107,7 @@ utils.exportEnvironmentFile = db => {
   @param {Date} date object
 */
 utils._generateBundleExportFileName = (envName, date=new Date()) => {
+  envName = envName || 'untitled-model';
```

Comment:
This is a fix for the generated filename that was introduced when we changed the new model name to be blank.